app.js

```diff
@@ -69,13 +69,20 @@ var app = express();
 //https://expressjs.com/en/advanced/best-practice-security.html
 //https://helmetjs.github.io/docs/
 const helmet = require('helmet');
-app.use(helmet());
+
+app.use(helmet({
```

Comment:
These headers are required in production environment. We should override these only when running locally.

Reply:
So should we have something .env variable to set production or development. Eg.
`const environment = process.env.ENVIRONMENT || 'production';`

Reply:
yes